portfolio.html

```diff
@@ -12,14 +12,15 @@
 <body>
 	<nav>
 		<div class="nav-link">
+			<a href="./portfoliofrancais.html" target="_blank"> <img src="france.png" alt="french"></a>
```

Comment:
Remove target="_blank" if you want to keep them on the same page and get the french version, makes sense to me.